app/src/main/java/io/manycore/robotcoder/demo/MainActivity.kt

```diff
@@ -0,0 +1,29 @@
+package io.manycore.robotcoder.demo
+
+import android.R
+import android.media.MediaPlayer
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+
+class MainActivity : AppCompatActivity() {
+
+    lateinit var mp:MediaPlayer
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        mp = MediaPlayer.create(this, R.raw.clip)
+        setContentView(R.layout.activity_main)
```

Comment:
MediaPlayer.create() must not launched from the UI thread. This will create performance issue.